src/components/autocomplete-dropdown/autocomplete-dropdown.test.tsx

```diff
@@ -49,59 +48,27 @@ describe('autocomplete-dropdown', () => {
   });
 
   it('it renders match suggestions', async () => {
-    findMatches = () => {
-      return [
-        {
-          id: 'result-first-id',
-          value: 'result-first-value',
-          route: 'result-first-route',
-        },
-        {
-          id: 'result-second-id',
-          value: 'result-second-value',
-          route: 'result-second-route',
-        },
-      ];
-    };
-
-    const wrapper = subject({ findMatches });
+    const searchResults = [
+      stubResult('first'),
+      stubResult('second'),
+    ];
```

Comment:
maybe if this used widely we can do a function?
```js
const getSearchResults = (n) => {
 const r = []
 return forLoop1ToN( r.push( stubResult(..) )
}
 ```

Reply:
Good idea. I was thinking about that too so I think you've given me a good enough reason to extract!